validator/src/main/resources/expected-data-template/otelFargateExpectedMetric.mustache

```diff
@@ -0,0 +1,234 @@
+-
+  metricName: pod_cpu_utilization_over_pod_limit
+  namespace: ContainerInsights
+  dimensions:
+    -
+      name: ClusterName
+      value: SKIP
```

Comment:
I think we can match the value for ClusterName. If we can't its fine. We just need to make sure the dimensions are there.